packages/api/src/controllers/webhooksController.ts

```diff
@@ -115,6 +126,59 @@ export default async function webhookController(fastify: FastifyInstance) {
     },
   );
 
+  fastify.withTypeProvider<TypeBoxTypeProvider>().post(
+    "/amazonses",
```

Comment:
Small nitpick, but can we rename this `amazon-ses`?

Reply:
Yep, it could also be `aws` if we'd like to avoid multiple words.

Reply:
Either `aws-ses`, `ses` or `amazon-ses` work for me!